executors/stable/requirements.txt

```diff
@@ -1,7 +1,5 @@
 numpy
 tqdm
-pytorch_lightning
-einops
-numpy
-omegaconf
-https://github.com/crowsonkb/k-diffusion/archive/master.zip
+jina
+docarray
+git+https://github.com/AmericanPresidentJimmyCarter/stable-diffusion.git
```

Comment:
We need to have a proper tag for the github repo since your library is at an early stage you will have to introduce breaking change at some point so we can't rely only on the master branch